### ISSUE TITLE
hotmail emails seem to go missing at a higher rate than other services

### ISSUE THREAD

**Issue by shane-tomlinson** [mozilla/fxa-auth-server#2506]
The title is purely based on anecdotal evidence from the number of bug reports we receive from Hotmail users who never receive FxA emails. When looking at supposed [delivery](https://sql.telemetry.mozilla.org/queries/56040/source#146958) and [bounce](https://analytics.amplitude.com/mozilla-corp/chart/fosrwwd) rates, hotmail isn't drastically different to other providers. Is the seemingly high number of hotmail reports imagined? Is there really a problem such as emails being sent and hotmail filtering them away w/o a bounce notification?

For some examples, see:
* https://bugzilla.mozilla.org/show_bug.cgi?id=1472619
* https://bugzilla.mozilla.org/show_bug.cgi?id=1459499

cc @irrationalagent, @davismtl, @philbooth

**Comment by shane-tomlinson** [comment]
@jrgm or @jbuck is there anything in the logs that would indicate that Hotmail emails are being lost but not reported as bounced?

**Comment by davismtl** [comment]
> The title is purely based on anecdotal evidence from the number of bug reports we receive from Hotmail users who never receive FxA emails.

Purely anecdotal too, I experienced the same thing at my lost job where I sent millions of emails per day. (FWW AOL is the worst)

@rfk and I had reached out to Microsoft last year (bug?) and they recommended we get ourselves whitelisted. Not all ESPs look at the whitelist but Microsoft recommended it. While we decided not to do it because it wasn't a standard, there may be benefit in doing it if we are still observing problems.
